packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -161,6 +171,16 @@ export class NextjsExportPodUtils {
     return { error: null };
   }
 
+  static async switchToBranch({
```

Comment:
It's a bit unintuitive that this method is called `switchToBranch` but the arguments are a path and a version.  

How come this method is never invoked anywhere in plugin-core? Does this functionality only apply to CLI flow for publishing right now?

Reply:
see the gdco. we are currently not using this process but want to adopt it to have versioned template updates